google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudConfigurable.java

```diff
@@ -153,9 +148,20 @@ public boolean isModified() {
 
   @Override
   public void apply() throws ConfigurationException {
-    configuration.setCloudProjectName(projectSelector.getText());
-    configuration.setCloudSdkExecutablePath(
-        CloudSdkUtil.toExecutablePath(cloudSdkDirectoryField.getText()));
+    if (StringUtil.isEmpty(cloudSdkDirectoryField.getText())
```

Comment:
How do they runtimeConfigurationErrors show up to the user? Some red writing on the form somewhere?